client/state/data-layer/wpcom-http/pipeline/remove-duplicate-gets/index.js

```diff
@@ -89,12 +87,16 @@ export const removeDuplicateGets = outboundData => {
 	}
 
 	const key = buildKey( nextRequest );
-	const queued = requestQueue.get( key );
+	const [ lastRequest, queued ] = requestQueue.get( key ) || [];
 	const request = addResponder( queued || { failures: [], successes: [] }, nextRequest );
+	const now = Date.now();
+	const shouldCull = lastRequest && now - lastRequest < hysteresis;
```

Comment:
definition of cull: `reduce the population of (a wild animal) by selective slaughter: he sees culling deer as a necessity | (as noun culling) : kangaroo culling.`

Reply:
yes, in the slaughter domain

in software it's generally associated with graphics…

> In 3D rendering the term culling describes the early rejection of objects of any kind (objects, draw calls, triangles and pixels) that don't contribute to the final image.

Reply:
slaughtered the wonderful beautiful fitting name in 89b3b2d :trollface: